crates/iroha_multisig_data_model/src/lib.rs

```diff
@@ -22,10 +37,10 @@ pub struct MultisigAccountArgs {
     /// </div>
     // FIXME #5022 prevent multisig monopoly
     // FIXME #5022 stop accepting user input: otherwise, after #4426 pre-registration account will be hijacked as a multisig account
-    pub account: PublicKey,
-    /// List of accounts and their relative weights of responsibility for the multisig
+    pub account: AccountId,
+    /// List of signatories and their relative weights of responsibility for the multisig account
     pub signatories: BTreeMap<AccountId, Weight>,
-    /// Threshold of total weight at which the multisig is considered authenticated
+    /// Threshold of total weight at which the multisig account is considered authenticated
     pub quorum: u16,
```

Comment:
```suggestion
    pub quorum: NonZeroU16,
```

Reply:
0 is not semantically well defined. Does it mean that transaction should be approved immediately on `MultisigPropose`?

Reply:
There is no difference between 0 and 1 in the sense of that the initial approval by proposal never executes the instructions even if the quorum is met. I probably should at least document this. Btw I could keep the previous behavior but didn't think it would be worth complicating the propose logic just for supporting "one shot to execute" cases. What do you think?